Source/FullScreenPokemon.ts

```diff
@@ -4400,7 +4400,8 @@ module FullScreenPokemon {
          *                   to optionally override the player's inventory.
          */
         openItemsMenu(settings: IItemsMenuSettings): void {
-            var items: IItemSchema[] = settings.items || this.ItemsHolder.getItem("items");
+            var items: IItemSchema[] = settings.items ||
+                this.ModAttacher.fireModEvent("onOpenItemsMenu", "Nuzlocke Challenge");
```

Comment:
No, don't make the core game rely on mods for functionality.

Instead, you can pass in the items to the mod and let the mod modify them.